Tests/RequestTests.cs

```diff
@@ -9,53 +9,231 @@ namespace Tests
     [TestFixture]
     public class RequestTests
     {
-        private ZendeskApi api = new ZendeskApi(Settings.Site, Settings.Email, Settings.Password);
+        private readonly ZendeskApi _api = new ZendeskApi(Settings.Site, Settings.Email, Settings.Password);
 
         [Test]
-        public void CanGetRequests()
+        public void CanGetAllRequests()
         {
-            var res = api.Requests.GetAllRequests();
+            var res = _api.Requests.GetAllRequests();
             Assert.True(res.Count > 0);
-        }        
+        }
+
+        [TestCase(1, 1)]
+        [TestCase(1, 2)]
+        public void CanGetAllRequestsPaged(int perPage, int page)
+        {
+            Assert.DoesNotThrow(() =>
```

Comment:
I can see what you are saying, but note the test will fail if any exception is thrown without the Assert.DoesNotThrow.

but if you want to leave it feel free to do so